static/js/partials/render.js

```diff
@@ -14,10 +14,11 @@ function newImages (images) {
 
       button.dataset.image = image[0]
       button.dataset.shot = 0
-
+// set comments here @jeroen
       const src = `../images/thumbnails/thumbnails_large/${image[0]}/${image[0]}_0.png`
       const img = element.image(src, '')
-
+// Tim placed a button inside the thing here so we can make ap preview button for each element
```

Comment:
Is this done?

Reply:
No, he is nog niet done yet

Reply:
```js
parentEl.appendChild(previewLink)
```

But I don't think having a `button` inside a `button` element is semantically correct.
Let's discuss this with @MartijnReeuwijk tomorrow.

Reply:
tru maybe make it an A

Reply:
Also styling with cach reminder